README.rst

```diff
@@ -12,6 +12,9 @@
 .. image:: https://codecov.io/gh/nipy/dipy/branch/master/graph/badge.svg
   :target: https://codecov.io/gh/nipy/dipy
 
+.. image:: https://api.codacy.com/project/badge/Grade/65d6b1c8dfb74f2d8401cd213f55d710
+  :target: https://www.codacy.com/app/Borda/dipy?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Borda/dipy&amp;utm_campaign=Badge_Grade
+
```

Comment:
This badge can be created later, we need to create a specific account for DIPY. So you can remove it for the moment. 
What do you think @arokem?